src/compiler/scala/tools/nsc/tasty/bridge/ContextOps.scala

```diff
@@ -553,9 +563,11 @@ trait ContextOps { self: TastyUniverse =>
       def reportParameterizedTrait(cls: Symbol, decls: u.Scope): Unit = {
         val traitParams = traitParamAccessors(cls).getOrElse(Iterable.empty)
         if (traitParams.nonEmpty) {
-          val parameters = traitParams.map(_.nameString)
-          val msg = s"parameterized trait ${parameters.mkString(s"${cls.nameString}(", ", ", ")")}"
-          unsupportedError(msg)(this.withOwner(cls.owner))
+          log {
+            val parameters = traitParams.map(_.nameString)
+            s"parameterized trait ${parameters.mkString(s"${cls.nameString}(", ", ", ")")}"
+          }
+          cls.updateAttachment(new u.DottyParameterisedTrait(traitParams.toList))
```

Comment:
perhaps theres a more efficient representation we can use for this

Reply:
or I guess more lightweight would be that its not necessary to record the actual parameters to present a correct error